apollo-backend/src/main/java/io/logz/apollo/controllers/StackController.java

```diff
@@ -0,0 +1,39 @@
+package io.logz.apollo.controllers;
```

Comment:
I don't understand this class. We have controller for stacks. This does not open rest, and uses the "LoggedIn" annotation. Remove it/convince me you need it

Reply:
Stack is an abstract class therefore I can't create a Stack object. When I'm calling getStack(), I need to know if it's an EnvironmentsStack or a ServicesStack to create the specific type. I think that the appropriate place for it is in a general StackController and not in EnvironmentsStackController or in ServicesStackController. Convincing enough?

Reply:
@mayhuberman no.. if its not open REST, its not a controller.
Either you open this as an API, or convert to StackService or something like that

Reply:
Got ya 👍